src/services.rs

```diff
@@ -195,6 +195,72 @@ pub enum ChannelKind {
     OpenClosed,
 
     //
+    // # String
+    //
+
+    /// The service is used to read or set the name associated with the service.
+    ///
+    /// # JSON - Note - perhaps this should be a tag instead?
+    ///
+    /// This kind is represented by the string "Name".
+    /// ```
+    /// use foxbox_taxonomy::services::*;
+    /// use foxbox_taxonomy::parse::*;
+    ///
+    /// let parsed = ChannelKind::from_str("\"Name\"").unwrap();
+    /// assert_eq!(parsed, ChannelKind::Name);
+    /// ```
+    Name,
+
+    /// The service is used to read or set the location associated with the
+    /// service. This is typically used to store the room within the house
```

Comment:
So far, we discussed the idea of putting this in a `tag`. What's the use case of putting it somewhere else?